src/PostgresExtensionsServiceProvider.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Fuwasegu\Postgres;
+
+use Fuwasegu\Postgres\Connectors\ConnectionFactory;
+use Illuminate\Database\DatabaseServiceProvider;
+use Illuminate\Database\DatabaseTransactionsManager;
+use Override;
+
+class PostgresExtensionsServiceProvider extends DatabaseServiceProvider
```

Comment:
Is this `extends` required?

Reply:
[f987a13](https://github.com/fuwasegu/laravel-pg-extensions-next/pull/2/commits/f987a132f716e6c0a3f9df3c6e48328785254cc3)